packages/mdc-list/adapter.ts

```diff
@@ -75,6 +75,12 @@ export interface MDCListAdapter {
    */
   isRootFocused(): boolean;
 
+  /**
```

Comment:
You also need to sync this to internal and implement this adapter before merging it.